test/functional/cmdLineTests/criu/src/org/openj9/criu/OptionsFileTest.java

```diff
@@ -293,6 +293,14 @@ static void jitOptionsTest(String[] args) {
 		System.out.println("Pre-checkpoint");
 		CRIUTestUtils.checkPointJVM(criuSupport, imagePath, true);
 		System.out.println("Post-checkpoint");
+
+		// Sleep to ensure that the JVM produces output needed for test success
+		int msToSleep = 2000;
+		try {
+			Thread.sleep(msToSleep);
```

Comment:
```suggestion
			Thread.sleep(2000);
```
I think `msToSleep` declaration can be skipped.